docs/default-firebase-data.json

```diff
@@ -610,7 +610,7 @@
       "title" : ""
     },
     "edilson_ngulele" : {
-      "bio" : "Edilson is an Android developer by passion. Enjoys coding, writing and talking about code. If he's not doing one of these, he's certainly reading about code or self-awareness.",
+      "bio" : "Edilson é um desenvolvedor Android por paixão. Gosta de programar, escrever e falar sobre código. Quando não estiver a fazer uma destas coisas, ele está certamente a ler sobre código ou autoconsciência.",
```

Comment:
@rosariopfernandes  maybe here you should consider replacing the verb _estiver_  by the verb _está_

_Quando não está a fazer uma destas coisas, ele está certamente a ler sobre código ou autoconsciência._